libvmi/driver/kvm/kvm.c

```diff
@@ -1364,6 +1380,17 @@ kvm_init_vmi(
     }
     vmi->num_vcpus = info.nrVirtCpu;
 
+#ifndef HAVE_LIBVMI_REQUEST
+    char *version = exec_info_version(kvm);
+    dbprint(VMI_DEBUG_KVM, "--Checking QEMU version string : %s", version);
+    // find substring "vmi"
```

Comment:
That works for me. There is a json-c dependency for Rekall profiles already, so adding that as a KVM dependency too would be fine.